src/BugBundle/Entity/User.php

```diff
@@ -219,4 +243,135 @@ public function removeRole(\BugBundle\Entity\Role $roles)
     {
         $this->roles->removeElement($roles);
     }
+
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    public $path;
+
+    public function getAbsolutePath()
```

Comment:
entity should not be responsible for files management, please decouple it to separate service